src/main/java/org/folio/fqm/service/FqlToSqlConverterService.java

```diff
@@ -1,15 +1,27 @@
 package org.folio.fqm.service;
 
-import org.folio.fql.model.*;
+import org.folio.fql.model.AndCondition;
+import org.folio.fql.model.ContainsCondition;
+import org.folio.fql.model.EmptyCondition;
+import org.folio.fql.model.EqualsCondition;
+import org.folio.fql.model.FieldCondition;
+import org.folio.fql.model.FqlCondition;
+import org.folio.fql.model.GreaterThanCondition;
+import org.folio.fql.model.InCondition;
+import org.folio.fql.model.LessThanCondition;
+import org.folio.fql.model.NotContainsCondition;
+import org.folio.fql.model.NotEqualsCondition;
+import org.folio.fql.model.NotInCondition;
+import org.folio.fql.model.RegexCondition;
 import org.folio.fql.service.FqlService;
-import org.folio.fqm.exception.ColumnNotFoundException;
+import org.folio.fql.service.FqlValidationService;
+import org.folio.fqm.exception.FieldNotFoundException;
 import org.folio.fqm.utils.SqlFieldIdentificationUtils;
 import org.folio.querytool.domain.dto.DateType;
 import org.folio.querytool.domain.dto.EntityDataType;
 import org.folio.querytool.domain.dto.EntityType;
-import org.folio.querytool.domain.dto.EntityTypeColumn;
+import org.folio.querytool.domain.dto.Field;
```

Comment:
I decided that it'd be clearer to use our Field as `Field` rather than Jooq's, since we refer to our `EntityType` and other objects directly.  This makes it a little more verbose to say when we're manipulating/returning a Jooq field, instead